migrations/v7.js

```diff
@@ -0,0 +1,135 @@
+  whereContent('Matching - where matching', async content => {
+    matchings = content.filter(({ _component }) => _component === 'matching');
+    return matchings.length;
+  });
+  mutateContent('Matching - add _isRandomQuestionOrder attribute', async content => {
+    matchings.forEach(matching => (matching._isRandomQuestionOrder = false));
+    return true;
+  });
+  checkContent('Matching - check _isRandomQuestionOrder attribute', async content => {
+    const isValid = matchings.every(matching => matching._isRandomQuestionOrder === false);
+    if (!isValid) throw new Error('Matching - _isRandomQuestionOrder attribute invalid');
+    return true;
+  });
+  updatePlugin('Matching - update to v7.3.0', { name: 'adapt-contrib-matching', version: '7.3.0', framework: '>=5.19.1' });
+});
+
+describe('Matching - v7.3.0 to v7.4.1', async () => {
+  whereFromPlugin('Matching - from v7.3.0', { name: 'adapt-contrib-matching', version: '<7.4.1' });
+  whereContent('Matching - where matching', async content => {
+    matchings = content.filter(({ _component }) => _component === 'matching');
+    return matchings.length;
+  });
+  mutateContent('Matching - add _hasItemScoring attribute if missing', async content => {
+    matchings.forEach(matching => {
+      if (!_.has(matching, '_hasItemScoringmatching')) matching._hasItemScoring = false;
+    });
+    return true;
+  });
+  mutateContent('Matching - add _allowOnlyUniqueAnswers attribute if missing', async content => {
+    matchings.forEach(matching => {
+      if (!_.has(matching, '_allowOnlyUniqueAnswers')) matching._allowOnlyUniqueAnswers = false;
+    });
+    return true;
+  });
+  checkContent('Matching - check _hasItemScoring attribute', async content => {
+    const isValid = matchings.every(matching => matching._hasItemScoring === false);
+    if (!isValid) throw new Error('Matching - _hasItemScoring attribute invalid');
+    return true;
+  });
+  checkContent('Matching - check _allowOnlyUniqueAnswers attribute', async content => {
+    const isValid = matchings.every(matching => matching._allowOnlyUniqueAnswers === false);
+    if (!isValid) throw new Error('Matching - _allowOnlyUniqueAnswers attribute invalid');
+    return true;
+  });
+  updatePlugin('Matching - update to v7.4.1', { name: 'adapt-contrib-matching', version: '7.4.1', framework: '>=5.19.1' });
+});
+
+describe('Matching - v7.4.1 to v7.5.0', async () => {
+  whereFromPlugin('Matching - from v7.4.1', { name: 'adapt-contrib-matching', version: '<7.5.0' });
+  whereContent('Matching - where matching', async content => {
+    matchings = content.filter(({ _component }) => _component === 'matching');
+    return matchings.length;
+  });
+  mutateContent('Matching - add _canShowCorrectness attribute', async content => {
+    matchings.forEach(matching => (matching._canShowCorrectness = false));
+    return true;
+  });
+  mutateContent('Matching - add globals correctAnswerPrefix attribute', async content => {
```

Comment:
Consider adding a check for existence of globals (see https://github.com/adaptlearning/adapt-contrib-narrative/blob/issue/318/migrations/v4.js#L20)